src/app/checkout/step-3/step-3.component.js

```diff
@@ -132,6 +137,8 @@ export default angular
       cartData: '<',
       submit: '<',
       onSubmitBtnChangeState: '&',
-      onSubmitted: '&'
+      onSubmitted: '&',
+      onSubmittingOrder: '&',
+      submittingOrder: '<'
```

Comment:
Do you think this would be better staying local (by not having the input)? It just took me a bit to figure why it was going out the output and coming right back in the input of the component. It is nice that the onSubmittingOrder call can set it everywhere with one call though. I'm fine with either.